ChatIntegrationExamples/CreatePrivateChat.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<DMSScript options="272" xmlns="http://www.skyline.be/automation">
+	<Name>Create Private Chat</Name>
+	<Description>Creates a private chat for a given user and the DataMiner Teams Bot in Microsoft Teams. Make sure to have a tenant set for your organization in the DCP Admin App (https://admin.dataminer.services).</Description>
```

Comment:
Could we change this from "DCP Admin App" to just "Admin App"? We've adjusted this in the documentation already, to avoid any unnecessary references to "DCP". This change should probably also be implemented in other locations outside the scope of the present pull request.

```suggestion
	<Description>Creates a private chat for a given user and the DataMiner Teams Bot in Microsoft Teams. Make sure to have a tenant set for your organization in the Admin App (https://admin.dataminer.services).</Description>
```